cop_prinstrat/code/CoP_Moderna_Run.R

```diff
@@ -0,0 +1,116 @@
+#Sys.setenv(TRIAL = "moderna_mock")
+
+#if (.Platform$OS.type == "windows") .libPaths(c("C:/Users/yhuang/Documents/renv/library/R-4.0/x86_64-w64-mingw32", "C:/Users/yhuang/AppData/Local/Temp/RtmpUPdelA/renv-system-library",
+#.libPaths()))
+#
+#
+#
+##----------------------------------------------- 
+## obligatory to append to the top of each script
+##renv::activate(project = here::here(".."))
+renv::activate(project = here::here())
+
+source(here::here("..", "_common.R"))
+#
+#
+## There is a bug on Windows that prevents renv from working properly. The following code provides a workaround:
+#if (.Platform$OS.type == "windows") .libPaths(c("C:/Users/yhuang/Documents/renv/library/R-4.0/x86_64-w64-mingw32", "C:/Users/yhuang/AppData/Local/Temp/RtmpUPdelA/renv-system-library",
+#.libPaths()))
+#
+#
+# population is either 57 or 29
+Args <- commandArgs(trailingOnly=TRUE)
+if (length(Args)==0) Args=c(pop="29")
+pop=Args[1]; myprint(pop)
+if(!has29 & pop=="29") {
+    print("Quitting because there are no Day 29 markers")
+    quit()
+} else if(!has57 & pop=="57") {
+    print("Quitting because there are no Day 57 markers")
+    quit()
+}
+
+save.results.to = paste0(here::here("output"), "/D", pop,"/");
+if (!dir.exists(save.results.to))  dir.create(save.results.to)
+print(paste0("save.results.to equals ", save.results.to))
+    
+#index1=1;index2=1
+
+library(kyotil)
+library(splines)
+library(nnet)
+
+#setwd("~/correlates_reporting")
+
+dat.mock<-read.csv(here::here("..","data_clean", "moderna_mock_data_processed.csv")) 
+  
+y.v<-"EventIndPrimaryD"%.%pop       
+
+if (pop=="57") {
+    dat.mock$wt.0=dat.mock$wt.D57
+    dat.mock$TwophasesampInd.0 = dat.mock$TwophasesampIndD57
+    dat.mock$ph1=dat.mock$ph1.D57   
+} else if (pop=="29") {
+    dat.mock$wt.0=dat.mock$wt.D29
+    dat.mock$TwophasesampInd.0 = dat.mock$TwophasesampIndD29 
+    dat.mock$ph1=dat.mock$ph1.D29
+} else stop("wrong pop")
+
+     
+dat.wide=subset(dat.mock, Bserostatus==0 & !is.na(wt.0))    
+     
+#
+#source(here::here("YingFunctions","NonparFun_NoW.R"))
+#
+source(here::here("YingFunctions","Fun_COVIDgeneral.R"))
+#
+#source(here::here("YingFunctions","function10_15_Full.R"))
+#
+source(here::here("YingFunctions","FunctionCall.R"))
```

Comment:
Is there a need to distinguish between `code` and `code/YingFunctions`? Or could all code be kept in one folder?